src/main/java/at/hannibal2/skyhanni/config/features/dungeon/DungeonConfig.java

```diff
@@ -99,9 +99,14 @@ public class DungeonConfig {
     @FeatureToggle
     public boolean highlightSkeletonSkull = true;
 
+    @Expose
+    @ConfigOption(name = "Chests Config", desc = "")
+    @Accordion
+    public DungeonChestConfig chest = new DungeonChestConfig();
+
     @Expose
     @ConfigOption(name = "Croesus Chest", desc = "Adds a visual highlight to the Croesus inventory that " +
-        "shows unopened chests.")
+        "shows unopened chests.") // TODO move to chest
```

Comment:
What do you want to move to chest category? the toggle `croesusUnopenedChestTracker`?

Reply:
Sorry, this time I was so lazy that I didn't event write out the whole config move (like last time). Written the configmove in the todo now. And this todo won't be resolved in this PR, it is just a notice that it should be moved.